public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -228,7 +228,7 @@ export class PanelQueryRunner {
         return of({
           ...data,
           state: LoadingState.Error,
-          error: toDataQueryError(err),
+          errors: [toDataQueryError(err)],
```

Comment:
this PR fixes the inspector so it also shows errors when the array is set... that is good because the single error flavor is deprecated in favor of using the array